.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,8 @@
+# Goal
+
+<!-- Description of what the PR is trying to accomplish. -->
```

Comment:
This comment wouldn't show up when creating a pull request would it? Goal is pretty clear but we might want to have an actual explanation on testing.

Reply:
It will appear in the edit box where you type in the description, but it wouldn't once you submit the PR